cdn-uploader.js

```diff
@@ -18,6 +18,7 @@ function UploadFilesFromPath(config) {
 			await client.useDefaultSettings();
 			console.log(`[+] Starting to upload the content of "${config.sourcePath}".`);
 			await client.ensureDir(`/${config.cpCode}/${config.projectName}`);
+			await client.clearWorkingDir()
```

Comment:
If I am not wrong, working dir is `${config.cpCode}/${config.projectName}`.

In this case, you will remove all versions. *Take care about it, you could delete published versions only with a PR*!

Reply:
Yeah I think you are right mmmmm